docs/about/cookies.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: Cookie Statement
 description: Explanation of Able Player's use of functional cookies
+layout: default
+parent: About this website
+nav_order: 3
 ---
 
-## 🍪 Cookie Statement
+# Cookie Statement
 
-Able Player uses cookies to enhance accessibility and improve user experience. These cookies are **strictly functional**—they help remember your preferences, such as playback settings, but do **not** track your activity or collect personal data.
+Able Player uses cookies to enhance accessibility and improve user experience. These cookies are **strictly functional**—they help remember your preferences, such as playback settings, but do **not** track your activity or collect personal data. 
```

Comment:
This should be about how wpaccessibility.org uses cookies. E.g. "`wpaccessibility.org` uses cookies in its media player, Able Player. Able Player uses cookies to..."